adoc/SAP_HA740_SetupGuide_SLE12.adoc

```diff
@@ -741,15 +669,16 @@ NOTE: Be careful there are two packages available. The package {s4sClConnector}
 continues to contain the old version 1.1.0 (SAP API 1). The package
 {s4sClConnector3} contains the new version 3.1.x (SAP API 3).
 The package {s4sClConnector3} with version 3.1.x implements the SUSE SAP API
-version 3. New features like SAP Rolling Kernel Switch (RKS) and the migration of ASCS are
-only supported with this new version.
+version 3. New features like SAP Rolling Kernel Switch (RKS) and the migration
+of ASCS are only supported with this new version.
 
 For the ERS and ASCS instance edit the instance profiles
 {mySid}_{myInstAscs}_{myVipNAscs} and {mySid}_{myInstErs}_{myVipNErs} in the
 profile directory _/usr/sap/{mySid}/SYS/profile/_.
 
-You need to tell the {sapStartSrv} to load the HA script connector library and
-to use the {s4sClConnector3}.
+You need to tell the `{sapStartSrv}` to load the HA script connector library and
+to use the `{s4sClConnector3}`. On the other hand, please make sure the feature
+_Autostart_ is *not* used.
```

Comment:
```
On the other hand, please make sure the feature
_Autostart_ is *not* used.
```
should be changed to:

Additionally, make sure that the feature _Autostart_ is *not* used.